File: leetcode/1_two_sum/two_sum.c
/**
 * time: 216 ms
 * memory: 7.7 MB
 */
int* twoSum(int* nums, int numsSize, int target, int* returnSize){
    int i = 0, j = 0;
    int *arr = malloc(2 * sizeof(int));
    for (i = 0; i < numsSize; i++) {
        for (j = i + 1; j < numsSize; j++) {
            if (nums[i] + nums[j] == target) {
                arr[0] = i;
                arr[1] = j;
                *returnSize = 2;
            }
        }
    }
    
    return arr;
}

File: leetcode/1_two_sum/two_sum.py
# runtime: 5440 ms
# memory: 14.8 MB
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        l = len(nums)
        for i in range(0, l):
            for j in range(i+1, l):
                if nums[i] + nums[j] == target:
                    return [i, j]
